avm/res/insights/action-group/README.md

```diff
@@ -25,11 +25,14 @@ The following section provides usage examples for the module, which were used to
 
 >**Note**: To reference the module, please use the following syntax `br/public:avm/res/insights/action-group:0.1.2`.
```

Comment:
This also needs to be fix with latest <version> approach

Reply:
done